src/components/ColumnModal/ColumnModal.tsx

```diff
@@ -0,0 +1,59 @@
+import { memo, useCallback, useState, useMemo } from 'react';
+import './ColumnModal.css';
+import { availableColumns } from './types/columnModal.types';
+import { formatColumnName } from '../../unitls/formatColumnName';
```

Comment:
Add the spell checking extension to you IDE to avoid such cases further
`unitls ->  utils`